identity-aware-proxy/IAPClient/IAPClient.cs

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright 2018 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// [START generate_iap_request]
+
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Security.Claims;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Json;
+using Microsoft.IdentityModel.Tokens;
+using Newtonsoft.Json;
+
+namespace GoogleCloudSamples
+{
+    class IAPClient
```

Comment:
Is this supposed to be internal?

Reply:
Rebecca, Jon and I discussed this a year or two ago and came to the following conclusion:
We do not use the keywords private/public/internal etc. in samples unless they are necessary to make the code behave.  The code is sample code, and these keywords only slow down the reader.

Reply:
I can stick with what you decided back then, but in my opinion this is not a good practice to omit access modifiers. Sample code should follow best practices and be very well written. People often tend to copy-paste the code and we should lead by example here, representing Google. Readers can learn not only the actual API usage you are showing but also tips on how to write quality code using the language of the sample. Having said that, maybe my comment is not relevant for interpreted languages like python, but it's relevant for C# or Java.